packages/conventional-changelog-core/test/test.js

```diff
@@ -1,48 +1,47 @@
 'use strict'
 const conventionalChangelogCore = require('../')
 const expect = require('chai').expect
-const mocha = require('mocha')
-const describe = mocha.describe
-const it = mocha.it
 const gitTails = require('git-tails').sync
-const shell = require('shelljs')
-const gitDummyCommit = require('git-dummy-commit')
 const through = require('through2')
 const Promise = require('pinkie-promise')
 const semver = require('semver')
 const betterThanBefore = require('better-than-before')()
 const preparing = betterThanBefore.preparing
-const mkdirp = require('mkdirp')
-const writeFileSync = require('fs').writeFileSync
+const fs = require('fs')
+const writeFileSync = fs.writeFileSync
 const path = require('path')
 const tmp = require('tmp')
+const rimraf = require('rimraf')
+const { gitInit, gitDummyCommit, exec } = require('../../../tools/test-tools')
 
 let dir = ''
 
 betterThanBefore.setups([
   function () { // 1
-    shell.config.resetForTesting()
-    shell.cd(__dirname)
     dir = process.cwd()
-    const tmpDir = tmp.dirSync().name
-    shell.mkdir(tmpDir)
-    shell.cd(tmpDir)
-    shell.mkdir('git-templates')
-    shell.exec('git init --initial-branch master --template=./git-templates')
-    writeFileSync('package.json', '{ "name": "conventional-changelog-core", "repository": { "type": "git", "url": "https://github.com/conventional-changelog/conventional-changelog-core.git" } }')
+    const tmpDir = tmp.dirSync()
+    process.chdir(tmpDir.name)
+    gitInit()
+    fs.writeFileSync('package.json', JSON.stringify({
+      name: 'conventional-changelog-core',
+      repository: {
+        type: 'git',
+        url: 'https://github.com/conventional-changelog/conventional-changelog-core.git'
+      }
+    }))
```

Comment:
this change is done just for readability, as its easier to see how package.json is going to look like